FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+
+    // Get the Find interface.
+    wil::com_ptr<ICoreWebView2Find> webView2Find;
+    CHECK_FAILURE(webView2_17->get_Find(&webView2Find));
+
+    // By default Find will use the default UI and highlight all matches. If you want different behavior
+    // you can change the SuppressDefaultDialog and ShouldHighlightAllMatches properties here.
+
+    // Start the find operation with a callback for completion.
+    CHECK_FAILURE(webView2Find->StartFind(
+        findConfiguration.get(),
+        Callback<ICoreWebView2FindOperationCompletedHandler>(
+            [this](HRESULT result, BOOL status) -> HRESULT
+            {
+                if (SUCCEEDED(result))
+                {
+                    // Optionally update UI elements here upon successful find operation.
+                }
+                else
+                {
+                    // Handle errors.
+                }
+                return S_OK;
+            }).Get()));
+
+    // End user interaction is handled via UI.
+    return true;
+}
+//! [ExecuteFindWithDefaultUI]
+```
+
+```cpp
+//! [ExecuteFindWithCustomUI]
+bool AppWindow::ExecuteFindWithCustomUI(const std::wstring& findTerm)
+{
+    auto findConfiguration = InitializeFindConfiguration(findTerm);
+    if (!findConfiguration)
+    {
+        return false;
+    }
+    // Query for the ICoreWebView2_17 interface to access the Find feature.
+    auto webView2_17 = m_webView.try_query<ICoreWebView2_17>();
+    CHECK_FEATURE_RETURN(webView2_17);
+
+    // Get the Find interface.
+    wil::com_ptr<ICoreWebView2Find> webView2Find;
+    CHECK_FAILURE(webView2_17->get_Find(&webView2Find));
+
+    // Opt for using a custom UI for the find operation.
+    CHECK_FAILURE(webView2Find->put_SuppressDefaultDialog(true));
+
+    // Start the find operation with callback for completion.
+    CHECK_FAILURE(webView2Find->StartFind(
+        findConfiguration.get(),
+        Callback<ICoreWebView2FindOperationCompletedHandler>(
+            [this](HRESULT result, BOOL status) -> HRESULT
+            {
+                if (SUCCEEDED(result) && status)
+                {
+                    // Optionally update UI elements here upon successful find operation.
+                }
+                else
+                {
+                    // Handle errors or unsuccessful find.
+                }
+                return S_OK;
+            }).Get()));
+
+    // Note: In this example, navigation through find results (FindNext/FindPrevious)
+    // and stopping the find operation (StopFind) are assumed to be handled by
+    // custom UI elements and user interaction, not directly in code here.
+    // User could then connect functions such as FindNext, FindPrevious, and StopFind
+    // to corresponding custom UI elements.
+
+    return true;
+}
+//! [ExecuteFindWithCustomUI]
+```
+#### .NET C#
+```csharp
+//! [ConfigureAndExecuteFindWithDefaultUI]
+async Task ConfigureAndExecuteFindWithDefaultUIAsync(string findTerm)
+{
+    try
```

Comment:
We shouldn't show try/catch in examples as it implies it's necessary. (And if it's necessary we should make it unnecessary.)

Reply:
Please fix. Only include try/catch in sample code if its actually required for the particular sample scenario (not just as a good idea / catch all)